tests/micro/zephyr/conftest.py

```diff
@@ -53,6 +56,24 @@ def zephyr_boards() -> dict:
 ZEPHYR_BOARDS = zephyr_boards()
 
 
+def qemu_boards(board: str):
```

Comment:
can we place test utils in a test_utils.py instead of conftest?

Reply:
done.